mmv1/products/apigee/api.yaml

```diff
@@ -363,3 +363,53 @@ objects:
         'Creating an environment':
           'https://cloud.google.com/apigee/docs/api-platform/get-started/create-environment'
       api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances.attachments/create'
+  - !ruby/object:Api::Resource
+    name: 'EnvgroupAttachment'
+    base_url: '{{envgroup_id}}/attachments'
+    create_url: '{{envgroup_id}}/attachments'
+    delete_url: '{{envgroup_id}}/attachments/{{name}}'
+    self_link: '{{envgroup_id}}/attachments/{{name}}'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: True
+        allowed:
+          - True
+          - False
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    input: true
+    description: |
+      An `Environment Group attachment` in Apigee.
+    parameters:
+     - !ruby/object:Api::Type::String
+        name: 'envgroupId'
+        description: |
+          The Apigee environment group associated with the Apigee environment,
+          in the format `organisations/{{org_name}}/envgroups/{{envgroup_name}}`.
```

Comment:
Sorry I didn't get it. Whats the change recommended here? @c2thorn

Reply:
To change the s->z in `organizations`. I put it in a format so that you can just use the "commit suggestion" button on the comment if you want to do it that way.